internal/orchestrator/status.go

```diff
@@ -25,13 +25,17 @@ import (
 
 func (or *orchestrator) GetStatus(ctx context.Context) (status *fftypes.NodeStatus, err error) {
 
+	orgKey := config.GetString(config.OrgKey)
+	if orgKey == "" {
+		orgKey = config.GetString(config.OrgIdentityDeprecated)
```

Comment:
Should we have a helper for getting the org key, which checks both the new and deprecated key? Seems like this pattern is repeated in a few places.

Reply:
helper, mock, and test updates added